src/shogun/regression/KrrNystrom.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Written (W) 2016 Fredrik Hallgren
+ * Copyright (C) 1999-2009 Fraunhofer Institute FIRST and Max-Planck-Society
+ */
+
+#ifndef _KRRNYSTROM_H__
+#define _KRRNYSTROM_H__
+
+#include <shogun/regression/KernelRidgeRegression.h>
+
+namespace shogun {
+
+class CKrrNystrom : public CKernelRidgeRegression
+{
+public:
+	MACHINE_PROBLEM_TYPE(PT_REGRESSION); // Seems need to be set for all classes
+
+	// Default constructor
```

Comment:
For later iterations: please make this doxygen comformant, so that class docs can be generated automatically, see http://www.shogun-toolbox.org/doc/en/3.0.0/classshogun_1_1CKernelRidgeRegression.html